app/components-react/pages/PlatformAppStore.tsx

```diff
@@ -1,15 +1,28 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState, useEffect, useRef } from 'react';
 import Utils from 'services/utils';
 import urlLib from 'url';
 import BrowserView from 'components-react/shared/BrowserView';
 import { GuestApiHandler } from 'util/guest-api-handler';
 import * as remote from '@electron/remote';
 import { Services } from 'components-react/service-provider';
+import { Button } from 'antd';
+import { EMenuItemKey } from 'services/side-nav';
+import { $t } from 'services/i18n';
 
 export default function PlatformAppStore(p: { params: { appId?: string; type?: string } }) {
-  const { UserService, PlatformAppsService, PlatformAppStoreService, NavigationService } = Services;
+  const {
+    UserService,
+    PlatformAppsService,
+    PlatformAppStoreService,
+    NavigationService,
+    HighlighterService,
+  } = Services;
+  const [highlighterInstalled, setHighlighterInstalled] = useState<boolean>(
+    HighlighterService.views.highlighterVersion !== '',
+  );
   const [platformAppsUrl, setPlatformAppsUrl] = useState('');
-
+  const [currentUrl, setCurrentUrl] = useState<string>('');
+  const browser = useRef<any>(null);
```

Comment:
the typing here should i think be `<Electron.BrowserView | null>`, but it also doesnt look used anywhere?